packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -406,7 +418,7 @@ export default function DataviewsPatterns() {
 					fields={ fields }
 					actions={ actions }
 					data={ data || EMPTY_ARRAY }
-					getItemId={ ( item ) => item.name }
+					getItemId={ ( item ) => item.name ?? item.id }
```

Comment:
Based on https://github.com/WordPress/gutenberg/pull/62927/files#r1658553606 there's always a `item.name`.

Reply:
There's not. See thread for reference.